lib/src/mock.dart

```diff
@@ -766,8 +766,8 @@ _InOrderVerification get verifyInOrder {
         if (allInvocations.isNotEmpty) {
           otherCalls = " All calls: ${allInvocations.join(", ")}";
         }
-        fail("Matching call #${tmpVerifyCalls
-            .indexOf(verifyCall)} not found.$otherCalls");
+        fail(
+            "Matching call #${tmpVerifyCalls.indexOf(verifyCall)} not found.$otherCalls");
```

Comment:
It looks like this is now longer than 80 chars, can you split the string at...  `} ""not found.`

Reply:
Done